packages/usa-layout-docs/src/styles/_usa-layout-docs.scss

```diff
@@ -1,17 +1,9 @@
 @use "uswds-core" as *;
 
-// Flexbox positioning to move sidenav below main content on small screens
 .usa-layout-docs__sidenav {
```

Comment:
Thinking about one of the points in the original issue about not having demo code leak into the system's CSS. I wonder if we could achieve the same results by using padding utility classes in the `usa-docs-inner.twig` file, rather than adding style rules via this class. This would enable us to remove these style rules entirely. 

_What do you think about switching to utility classes so that we can remove `usa-layout-docs.scss` entirely?_

This is likely more a philosophical question than a practical one, since the impact on the CSS is so small. But it feels worth considering. Curious what you think!

Reply:
That seems like a good follow-up issue/enhancement.

I hesitate to make it in the scope of this PR because new guidance would be a requirement.